tests/foreman/api/test_subnet.py

```diff
@@ -481,7 +481,6 @@ def test_positive_delete_subnet_overridden_parameter_host_impact(self):
         CaseLevel: Integration
         """
 
-    @stubbed()
     @tier1
```

Comment:
Can we consider this as tier 2 test as there are two entities are involved? @vikask18 @mirzal  what you think?

Reply:
Tier1 is ok as we are validating the ```GET``` operation.

Reply:
IMO Tier 1, since we are working with only single entity ie. subnet. 
Only parameters are being added.